<?xml version="1.0"?>
<launch>
  <arg name="verbose"         value="true"       />
  
  <!-- FOR setup the env -->
  <param    name="pkg_name"   value="agile_apps" />
  <rosparam command="load"    file="$(find agile_apps)/config/yaml/leg_robot.yaml" />
 
  <!-- Load hardware interface -->
  <node name="leg_robot" pkg="agile_apps" type="leg_robot" output="screen" >
    <param name="verbose"     value="$(arg verbose)" />
    <param name="namespaces"  value="leg_robot"      />
  </node>

</launch>

